src/components/cardItem/CardItem.css

```diff
@@ -0,0 +1,17 @@
+/*creates the card*/
+.pokemon-card {
+    border: 1px solid #ccc;
+    border-radius: 12px;
+    padding: 16px;
+    margin: 16px;
+    width: 250px;
```

Comment:
Als je de cards in een grid zet, hoeven ze geen vaste breedte mee te krijgen (die zet je immers in de grid)